network-api/networkapi/wagtailpages/pagemodels/blog/blog_index.py

```diff
@@ -44,6 +44,38 @@ def __str__(self):
         return self.page.title + '->' + self.blog.title
 
 
+class FeaturedVideoPost(WagtailOrderable, models.Model):
+    page = ParentalKey(
+        'wagtailpages.BlogIndexPage',
+        related_name='featured_video_post',
+    )
+
+    blog_page = models.ForeignKey(
+        'wagtailpages.BlogPage',
+        on_delete=models.CASCADE,
+        related_name='+'
+    )
+    video_url = models.URLField(
```

Comment:
**Suggestion** I believe the copy process could be simplified for editors by using the [`embed` template tag](https://docs.wagtail.org/en/stable/advanced_topics/embeds.html#embed-tag) to convert the oembed URL to the Video URL.

Reply:
Updated to use oembed. great suggestion!